authentication/MediaWikiApiAuthentication.js

```diff
@@ -87,16 +84,16 @@ MWApiAuthenticator.prototype = {
 
 	doRequest: function (url, requestMethod, requestBody) {
 		var msg,
-			requestUri = new URI(url, false);
+			requestUri = new URI(url, false),
```

Comment:
Comma?

Reply:
Yeah, this is only one statement declaring 3 variables.